com.unity.render-pipelines.universal/Shaders/Utils/StencilDeferred.shader

```diff
@@ -121,12 +121,27 @@ Shader "Hidden/Universal Render Pipeline/StencilDeferred"
     TEXTURE2D_X_HALF(_GBuffer0);
     TEXTURE2D_X_HALF(_GBuffer1);
     TEXTURE2D_X_HALF(_GBuffer2);
+
+#if _RENDER_PASS_ENABLED
+    FRAMEBUFFER_INPUT_HALF(0);
+    FRAMEBUFFER_INPUT_HALF(1);
+    FRAMEBUFFER_INPUT_HALF(2);
+    FRAMEBUFFER_INPUT_FLOAT(3);
```

Comment:
Consider adding defined for the GBuffer FB Attachment indices, this is more meaninful and easier to understand.

Reply:
that definitely makes sense, will fix tomorrow